contracts/utils/structs/BitMaps.sol

```diff
@@ -4,7 +4,7 @@ pragma solidity ^0.8.20;
 
 /**
  * @dev Library for managing uint256 to bool mapping in a compact and efficient way, provided the keys are sequential.
- * Largely inspired by Uniswap's https://github.com/Uniswap/merkle-distributor/blob/master/contracts/MerkleDistributor.sol[merkle-distributor].
+ * Largely inspired by Uniswap's https://github.com/Uniswap/merkle-distributor/blob/25a79e8ec8c22076a735b1a675b961c8184e7931/contracts/MerkleDistributor.sol[merkle-distributor].
```

Comment:
Similar comment

This was written in june 2021 in https://github.com/OpenZeppelin/openzeppelin-contracts/pull/2710/commits/69e7005a70d314996ea8395e47fb957109d7b448, over 3 years ago. I would not touch that with a commit hash that is posterior to when the code was written.